src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -99,6 +99,9 @@ public partial struct AssemblyDefinition
         public System.Reflection.Metadata.StringHandle Name { get { throw null; } }
         public System.Reflection.Metadata.BlobHandle PublicKey { get { throw null; } }
         public System.Version Version { get { throw null; } }
+#if !NETSTANDARD11
+        public System.Reflection.AssemblyName GetAssemblyName() { throw null; }
```

Comment:
@safern, I had an offline conversation with @tmat regarding our recent experimentation when we try to add GetAssemblyName API in netstandard1.1.

It seems that it should be fine to just add this API to NS2.0